src/main/java/com/github/onsdigital/babbage/api/endpoint/content/Resource.java

```diff
@@ -23,13 +31,52 @@
     @GET
     public void get(@Context HttpServletRequest request, @Context HttpServletResponse response) throws IOException {
         try {
-            ContentResponse contentResponse = ContentClient.getInstance().getResource(request.getParameter("uri"));
+            String uri = request.getParameter("uri");
+            String width = request.getParameter("width");
+
+            ContentResponse contentResponse = ContentClient.getInstance().getResource(uri);
             String contentDispositionHeader = "inline; ";
             contentDispositionHeader += contentResponse.getName() == null ? "" : "filename=\"" + contentResponse.getName() + "\"";
             response.setHeader("Content-Disposition", contentDispositionHeader);
-            new BabbageContentBasedBinaryResponse(contentResponse, contentResponse.getDataStream(), contentResponse.getMimeType()).apply(request, response);
+
+            InputStream contentResponseBody = contentResponse.getDataStream();
+            BufferedImage image = null;
+
+            if (width != null) {
+
+                try {
+                    Integer w = Integer.parseInt(width, 10);
+
+                    if (uri.toLowerCase().endsWith(".png") || uri.toLowerCase().endsWith(".jpg") || uri.toLowerCase().endsWith(".jpeg")) {
+                        image = ImageIO.read(contentResponseBody);
```

Comment:
Need to check the api doc but this sounds like something that might also need to be closed.